docs/user-guide/mc.ipynb

```diff
@@ -1,885 +0,0 @@
-   "execution_count": null,
-   "id": "8975a81d-7b8b-41f5-9793-78208f6701d8",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "pos_sampler = mc.NormalDenseSampler(pos)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "e30f3cf8-01bc-44c2-a3e6-aa59caa47cff",
-   "metadata": {},
-   "source": [
-    "Next, we need to define how to collect the replicas and compute an output statistic.\n",
-    "In this case, we simply want to compute the mean and variance which is implemented by [VarianceAccum](../generated/modules/scippuncertainty.mc.accumulator.VarianceAccum.rst)."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "f20e9313-c42e-41d0-a72d-82204a183bc8",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "accumulator = mc.VarianceAccum()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "35d68fb7-5456-4842-aaf2-1162cf504900",
-   "metadata": {},
-   "source": [
-    "Finally, we can use [mc.run](../generated/modules/scippuncertainty.mc.driver.run.rst) to put everything together and actually perform the MC computation.\n",
-    "\n",
-    "We pass a `dict` as the samplers that identifies the `pos_sampler` with the `pos` argument of `compute_speed`.\n",
-    "The accumulators `dict` defines how to accumulate each output of `compute_speed`.\n",
-    "There is only one here, but the accumulators still have to match the name in the `dict` returned by `compute_speed`.\n",
-    "Since `time` has no uncertainties, we simply bind our fixed values using `partial`.\n",
-    "\n",
-    "`n_samples` corresponds to the number of replicas $R$.\n",
-    "It is very high in this case because the computation is quite fast.\n",
-    "In practice, numbers in the hundreds or low thousands are more feasible.\n",
-    "Lastly, we disable progress reporting as this does not work reliably in Jupyter."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "a02f4f2e-e4ea-4ab5-a643-b89b0dfcead8",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "from functools import partial\n",
-    "\n",
-    "results = mc.run(\n",
-    "    partial(compute_speed, time=time),\n",
-    "    samplers={\"pos\": pos_sampler},\n",
-    "    accumulators={\"speed\": accumulator},\n",
-    "    n_samples=100_000,\n",
-    "    progress=False,\n",
-    ")\n",
-    "speed_mc = results[\"speed\"]"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "b0dd1700-0f27-42dc-bce8-729afa1223e0",
-   "metadata": {},
-   "source": [
-    "Now compare the results of the two calculations.\n",
-    "It looks like MC and 'regular' uncertainty propagation are in agreement."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "e7056e81-2da3-4a2f-bcf1-45cc969a2c4c",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
-   "source": [
-    "pp.plot({\"regular\": speed_regular, \"mc\": speed_mc},\n",
```

Comment:
For this plot and the one below, it is pretty difficult to see any differences between the two curves. I think it would be useful to add a panel below the main canvas with residual differences? (like we did in the paper).

As you currently have to make the extra panel by hand, maybe we should have a helper function that does this in plopp? Something like `pp.residual_plot(a, b)`.
Some alternative syntax could be to operate on the plot after it's created, such as `pp.plot(a, b).residuals(a-b)`?

Reply:
Good idea. Done.